lib/modules/platform/codecommit/iam-client.ts

```diff
@@ -30,23 +19,10 @@ export async function getUserArn(): Promise<string> {
   const cmd = new GetUserCommand({});
   let res;
   try {
-    const userRes: GetUserCommandOutput = await iam.send(cmd);
+    const userRes = await iam.send(cmd);
     res = userRes?.User?.Arn;
   } catch (err) {
-    const match = userRe.exec(err.message);
-    if (match) {
-      logger.warn(
-        'It is recommended to add "IAMReadOnlyAccess" policy to this IAM user'
-      );
-      res = match.groups?.arn;
-    }
-    if (!res) {
-      logger.warn(
-        'Failed to get IAM user info, Make sure your user has "IAMReadOnlyAccess" policy'
-      );
-      throw err;
-    }
+    logger.debug('Failed to get IAM user info');
```

Comment:
Do we now expect it to succeed each time?

Reply:
yes, it will continue, it's not a deal breaker to filter by ARN author we can always get all open PRs